services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -7958,41 +7958,6 @@ void setUpMetaStoreAndMayProduceSnapshot(String clusterName, String regularStore
         "Wrote value schemas to meta system store for venice store: {} in cluster: {}",
         regularStoreName,
         clusterName);
-    // replica status for all the available versions
```

Comment:
Hmm, just to confirm, FC don't need this information anymore?

Reply:
That's right. FC does not need  meta stores anymore